source/_integrations/overkiz.markdown

```diff
@@ -52,21 +52,21 @@ Over 6000 devices from 60 brands are compatible with the Overkiz platform. This
 
 ### Z-Wave, Hue and Sonos not supported
 
-Even though most Overkiz hubs supports adding Z-Wave, Hue and Sonos devices, this isn't supported in the Overkiz integration. All these platforms have native integrations in Home Assistant which are more stable and feature rich.
+Even though most Overkiz hubs support adding Z-Wave, Hue, and Sonos devices, this isn't supported in the Overkiz integration. All these platforms have native integrations in Home Assistant which are more stable and feature-rich.
 
 ### Overkiz API limits
 
 **Execution queue is full on gateway: #xxxx-xxxx-xxxx (soft limit: 10)**
```

Comment:
In my review I mentioned this is a bug, and therefore should be fixed, not documented.

Reply:
I will remove this part. We already implemented backoff (exponentially) where we try a maximum of 10 times (to not overload the Somfy server with old requests) to fix this issue.

I kept it in here, since command execution will be delayed if you run more than 10 commands at the same time.

Reply:
@bdraco sorry, I couldn't reply to your comment on the other PR since that one is locked. I will reach out to you on Discord. In my message above there is a bit more context about the issue / flaw in Somfy's server.

Reply:
Sorry for the late follow up. I have addressed your feedback in 39000d2, and rebased on next.